src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -7,7 +7,7 @@ sidebarDepth: 2
 
 Ethereum is a distributed network of computers (known as nodes) running software that can verify blocks and transaction data. The software application, known as a client, must be run on your computer to turn it into an Ethereum node.
 
-**Note: it is not possible to run an execution client on its own anymore. After [The Merge](/upgrades/merge), both execution and consensus clients must be run together in order for a user to gain access to the Ethereum network.**
+**Note: it is not possible to run an execution client on its own anymore. After [The Merge](/upgrades/merge), both execution and consensus clients must be run together in order for a user to gain access to the Ethereum network. However, a consensus client can still run independently.**
```

Comment:
Is there a reason for adding this @Olshansk? AFAIK both clients are requires at this point.

As evidence, in order for the consensus client to even know _who_ the validators are, it currently relies on the records of the staking deposit contract which is part of the execution layer.
```suggestion
**Note: it is not possible to run an execution client on its own anymore. After [The Merge](/upgrades/merge), both execution and consensus clients must be run together in order for a user to gain access to the Ethereum network.**
```

Reply:
@wackerow I think the point is that you can run a node as a service instead. I also disagree with adding this; it's confusing and feels like it's trying to push centralized services.